### ISSUE TITLE
render AuxiliaryPreview only

### ISSUE THREAD

**Issue by fukemy** [dominicstop/react-native-ios-context-menu#51]
Hi, for some messages that can not took any actions(only accept reactions), but i still want to show the AuxiliaryPreview for this message.

But when the actionList is empty, the context menu not show, can u help?

**Comment by fukemy** [comment]
hi @dominicstop , can u help me to give some advices?
Like this message

![Screen Shot 2022-08-26 at 13 40 35](https://user-images.githubusercontent.com/8202437/186838816-319e3631-e98d-41e1-bf82-0a28df91324d.png)

It's contain delete options, so it can show AuxiliaryPreview

When message does not contain any actions, the context menu does not trigger, but I still want to show AuxiliaryPreview

**Comment by fukemy** [comment]
In my case, menuItems isEmpty, but I want to render ***menuTitle*** and *** AuxiliaryPreview***

**Comment by dominicstop** [comment]
hi unfortunately you need to have at least 1 menu item for aux. preview to work (this is a limitation w/ iOS's API) 

however, there is a workaround ✨

take a look at: [context menu preview example 08](https://github.com/dominicstop/react-native-ios-context-menu#contextmenuview-example-11), [aux. preview example 08](https://github.com/dominicstop/react-native-ios-context-menu#contextmenuview-auxiliary-preview---example-08), and [aux. preview example 09](https://github.com/dominicstop/react-native-ios-context-menu#contextmenuview-auxiliary-preview---example-09)

what they all have in common is that they don't have any menu items + they're using a custom preview.

(i'm sorry, I don't have access to my computer atm, so i apologize for the crude code example)

```
<ContextMenuView
  previewConfig={{
    previewType: hasNoMenuItems
      ? 'CUSTOM'
      : 'DEFAULT'
  }}
  renderPreview={() => (
    <CompToShowInContextMenu/>
  )}
/>
  <CompToShowInContextMenu/>
<ContextMenuView>
```

basically, if you know your context menu view comp. does not have any menu items, then just use a custom preview that just renders the same thing you put inside `ContextMenuPreview`

hope this helps you, xx

**Comment by fukemy** [comment]
Thanks, i got it

**Comment by prsvic** [comment]
Hello! using this workaround, is there a way for preview to stay in place instead of moving to the center of the screen?
